scene/gui/file_dialog.cpp

```diff
@@ -756,7 +754,7 @@ void FileDialog::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_unhandled_input"), &FileDialog::_unhandled_input);
 
 	ClassDB::bind_method(D_METHOD("_tree_selected"), &FileDialog::_tree_selected);
-	ClassDB::bind_method(D_METHOD("_tree_db_selected"), &FileDialog::_tree_dc_selected);
+	ClassDB::bind_method(D_METHOD("_tree_dc_selected"), &FileDialog::_tree_dc_selected);
```

Comment:
Any clue about what `dc` means in this method? Maybe it should be made more explicit.

Reply:
"double click", it's tied to the Tree's "item_activated" signal. Should it be renamed "_tree_double_clicked"?